### DISCUSSION TITLE
point_0697

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#249] [Roy]
"20200207T030909_20200207T031314_T50TMK"
![image](https://user-images.githubusercontent.com/54947195/147513925-148319de-a1a9-4458-8917-c9a0688aadee.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20200207T030909_20200207T031314_T50TMK"
![Captura](https://user-images.githubusercontent.com/86502862/152891838-9cf22007-62c3-4090-96e1-7be0352d0a42.JPG)

**Comment by AntoniCesar** [comment]
"20200618T025551_20200618T030100_T50TM
![Captura](https://user-images.githubusercontent.com/86502862/152892000-5967c90d-cc02-4147-96db-375727dd419e.JPG)
K"
